src/libkomwm.py

```diff
@@ -283,6 +277,8 @@ def print_warning(msg):
 def store_visibility(cl, dr_type, object_id, zoom):
     if object_id == '::default':
         object_id = ''
+    if type(dr_type) is str:
+        dr_type = (dr_type, None)
```

Comment:
How it can be not a string?

Reply:
it can be a tuple for cases when auto_comment is passed along

Reply:
I've changed the func's signature to make it more explicit.